syft/frameworks/torch/hook/hook.py

```diff
@@ -122,10 +131,14 @@ def __init__(
             # be agnostic to the means by which workers communicate (such as
             # peer-to-peer, sockets, through local ports, or all within the
             # same process)
-            self.local_worker = VirtualWorker(hook=self, is_client_worker=is_client, id="me")
+            self.local_worker = VirtualWorker(
```

Comment:
Wondering if we should rename `self.local_worker` to just `self.worker`, because that would make it possible to rename `self._virtual_workers` below to `self.local_workers`, which seems potentially more accurate.

Reply:
I feel that the current naming scheme seems to be more appropriate though. (@LaRiffle can we have your thoughts on this)

local_worker ->  Worker owned by the local machine
virtual_worker -> Machine present elsewhere but "virtually" present by ```alice = sy.VirtualWorker(hook, id="alice")```

Reply:
All VirtualWorkers are local but not all VirtualWorkers are virtual. Something seems off about that. Not saying you should fix it in this PR, just saying this PR is a bit confusing due to underlying naming issues.